scripts/scil_tractogram_convert_trk_to_hdf5.py

```diff
@@ -0,0 +1,108 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+
+"""
+Save connections as TRK to HDF5.
+
+This script is useful to convert a set of connections or bundles to a single
+HDF5 file. The HDF5 file will contain a group for each input file, with the
+streamlines stored in the specified space and origin (keep the default if you
+are going to use the connectivity scripts in scilpy).
+
+To make a file compatible with scil_tractogram_commit.py or
+scil_connectivity_compute_matrices.py you will have to follow this nomenclature
+for the input files:
+in_dir/
+    |-- LABEL1_LABEL1.trk
+    |-- LABEL1_LABEL2.trk
+    |-- [...]
+    |-- LABEL90_LABEL90.trk
+The value of first labels should be smaller or equal to the second labels.
+Connectivity scripts in scilpy only consider the upper triangular part of the
+connectivity matrix.
+
+By default, ignores the empty connections. To save them, use --save_empty.
+Note that data_per_point is never included.
+"""
+
+import argparse
+import logging
+import os
+
+from dipy.io.stateful_tractogram import Space, Origin
+from dipy.io.utils import is_header_compatible
+import h5py
+
+from scilpy.io.hdf5 import (construct_hdf5_header,
+                            construct_hdf5_group_from_streamlines)
+from scilpy.io.streamlines import load_tractogram_with_reference
+from scilpy.io.utils import (add_overwrite_arg, add_verbose_arg,
+                             assert_inputs_exist,
+                             assert_outputs_exist)
+
+
+def _build_arg_parser():
+    p = argparse.ArgumentParser(
+        description=__doc__, formatter_class=argparse.RawTextHelpFormatter)
+    p.add_argument('in_bundles', nargs='+',
+                   help='Path of the input connection(s) or bundle(s).')
+    p.add_argument('out_hdf5',
+                   help='Output HDF5 filename (.h5).')
+
+    p.add_argument('--stored_space', choices=['rasmm', 'voxmm', 'vox'],
+                   default='vox',
+                   help='Space in which the streamlines are stored '
+                        '[%(default)s].')
+    p.add_argument('--stored_origin', choices=['nifti', 'trackvis'],
+                   default='trackvis',
```

Comment:
I don't understand this. "Origin in which streamlines are stored" and why trackvis is the default.

Reply:
I changed to 'Voxel origin convention' and the default is "trackvis" because we mostly work with TRK and that's their convention.

(The --space and --origin default are made to work with scil_tractogram_convert_hdf5_to_trk.py so the default of this script is what is expected by scil_tractogram_convert_hdf5_to_trk.py)